### PR TITLE
Closes #120 qs update ophtha

### LINKED ISSUE #120: Update QS data for ophthamology
New test data required for Ophthalmology specific Questionnaire.

See https://github.com/pharmaverse/admiralophtha/blob/devel/dev/vfq.R

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! 

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. 

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by millerg23** [comment]
@bms63 this is the test Ophthamology Questionnaire that is appended to CDISC questionnaire. I also updated the label for the new variable `AE.AELAT` added in previous pull request.

**Comment by bms63** [comment]
Unsure if we can use, but just FYI @bundfussr @kaz462 @DeclanHodges

File: dev/update_ae.R [hunk 1]
```diff
@@ -18,7 +18,8 @@ ae$AELAT <- if_else(ae$AESOC == "EYE DISORDERS",
   apply(ae, 1, function(x) sample(lat, 1)),
   NA_character_
 )
-admiral_ae <- ae
+admiral_ae <- ae %>%
+  add_labels(AELAT = "Laterality")
```
Comment (bms63) [inline dev/update_ae.R:22]: You should do a `library(metatools)` for posterity :)

File: dev/update_qs.R [hunk 2]
```diff
@@ -0,0 +1,107 @@
+#' Program Name : update_qs.R
```
Comment (bms63) [inline dev/update_qs.R:1]: Lets remove this

File: dev/update_qs.R [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+#' Description : create VFQ questionnaire test data for ADMIRAL Ophtha
+
+#' Input : raw_qs
+
+#' Author : Ritika Aggarwal
```
Comment (bms63) [inline dev/update_qs.R:7]: Shall we just add this person in the DESCRIPTION file and drop this line?  I think the input can be removed as well.

Just trying to standardize these files.

File: dev/update_qs.R (hunk 2 above)
Comment (millerg23) [reply dev/update_qs.R]: Done.

File: dev/update_qs.R (hunk 3 above)
Comment (millerg23) [reply dev/update_qs.R]: Done. Think it fine to leave name out of DESCRIPTION file.

File: dev/update_ae.R (hunk 1 above)
Comment (millerg23) [reply dev/update_ae.R:23]: DONE.

**Pull request merged**